exchange-v2/test/v2/EIP712MetaTransaction.test.js

```diff
@@ -0,0 +1,689 @@
+const { deployProxy, upgradeProxy } = require('@openzeppelin/truffle-upgrades');
+const ExchangeSimpleV2 = artifacts.require("ExchangeSimpleV2.sol");
+const ExchangeSimpleV2_MetaTx = artifacts.require("ExchangeSimpleV2_MetaTx.sol");
+const ExchangeV2 = artifacts.require("ExchangeV2.sol");
+
+const TestRoyaltiesRegistry = artifacts.require("TestRoyaltiesRegistry.sol");
+const TransferProxyTest = artifacts.require("TransferProxyTest.sol");
+const ERC20TransferProxyTest = artifacts.require("ERC20TransferProxyTest.sol");
+const TestERC20 = artifacts.require("TestERC20.sol");
+
+const web3Abi = require('web3-eth-abi');
+const sigUtil = require('eth-sig-util');
+const { Order, Asset, sign } = require("../order");
+const { ETH, ERC20, ERC721, ERC1155, enc, id } = require("../assets");
+const { expectThrow } = require("@daonomic/tests-common");
+const truffleAssert = require('truffle-assertions');
+
+const ZERO = "0x0000000000000000000000000000000000000000";
+let ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
+let publicKey = "0x726cDa2Ac26CeE89F645e55b78167203cAE5410E";
+let privateKey = "0x68619b8adb206de04f676007b2437f99ff6129b672495a6951499c6c56bc2fa6";
+
+let executeMetaTransactionABI = {
```

Comment:
can we import abis from another file to make it more readable?

Reply:
agree. do it